src/ittnotify/jitprofiling.c

```diff
@@ -112,6 +114,28 @@ ITT_EXTERN_C iJIT_IsProfilingActiveFlags JITAPI iJIT_IsProfilingActive()
     return executionMode;
 }
 
+#if ITT_PLATFORM==ITT_PLATFORM_WIN
+int isPathRelative(char *path)
```

Comment:
Why we can't use default windows function `PathIsRelativeA`?

If we really need our own implementation lets change it a bit:
- add `static` keyword to not pull this function somewhere outside of this file
- rename function to `isValidAbsolutePath`
- adapt function code to return 1(true) if this is valid absolute path and 0(fasle) otherwise

Reply:
We can use default windows function `PathIsRelativeA` but then we will have to add and link library `shlwapi.lib`([API Link](https://learn.microsoft.com/en-us/windows/win32/api/shlwapi/nf-shlwapi-pathisrelativea)). 
This will simplify the work but I was not sure what might be the implications for adding a new library just for a single function.

Reply:
Ok then please change this function as I described above